source/imaer-gml/src/main/java/nl/overheid/aerius/gml/v5_0/togml/Result2GML.java

```diff
@@ -110,7 +111,7 @@ private AbstractCalculationPoint determineSpecificType(final CalculationPoint ae
     return returnPoint;
   }
 
-  private AbstractCalculationPoint fromNslCalculationPoint(final nl.overheid.aerius.shared.domain.v2.point.NSLCalculationPoint aeriusNSLPoint) {
+  private AbstractCalculationPoint fromNslCalculationPoint(final CIMLKCalculationPoint aeriusNSLPoint) {
```

Comment:
looks like a leftover NSL in `aeriusNSLPoint` (I'd suggest just using `aeriusPoint`)